src/main/java/com/seeroo/rentalservice/services/impl/WelcomeServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package com.seeroo.rentalservice.services.impl;
+
+import com.seeroo.rentalservice.dto.UserDetailsDTO;
+import com.seeroo.rentalservice.services.WelcomeService;
+import org.springframework.stereotype.Service;
+
+@Service
+public class WelcomeServiceImpl implements WelcomeService {
+    static String cIndia = "India";
+    static String myCompany = "Seeroo";
+
+    @Override
+    public String getWelcomeMessageBasedOnCountry(UserDetailsDTO userDetailsDTO) {
+        String welcomeMessage = "";
+        if (userDetailsDTO.getLocation().equals(cIndia)) {
```

Comment:
Compare the strings always with constants first 

```cIndia.equals(userDetailsDTO.getLocation())```

Explanations below

https://stackoverflow.com/questions/36501926/java-string-comparison-best-practice